app/assets/stylesheets/utils/_custom.scss

```
@@ -49,3 +49,16 @@
     letter-spacing: 1px;
     max-width: 250px
 }
+
+.warning-text__assistive {
```

```suggestion
.govuk-warning-text__text .warning-text__assistive {
```

It's best practice not to over-use `!important`, we can avoid this by adding this class `.govuk-warning-text__text` before `.warning-text__assistive`. Like the suggestion above

---

Done. Making use of govuk-visually-hidden as suggested instead of warning-text__assistive